### PR TITLE
ASU-1597 | Audit logger Elastic Search integration

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

File: audit_log/models.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from django.db import models
 from django.db.models import JSONField
+from django.utils.translation import gettext_lazy as _
 
 
 class AuditLog(models.Model):
     message = JSONField()
+    is_sent = models.BooleanField(default=False, verbose_name=_("is sent"))
```
Comment (tuomas777) [inline audit_log/models.py:8]: What about using a nullable DatetimeField (like `sent_at`) for this, which would hold a bit more perhaps useful information for debugging etc with ~zero cost?

File: audit_log/management/commands/clear_audit_log_entries.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = (
+        "Clear AuditLog which is already sent to Elasticsearch,"
+        "only clear if settings.CLEAR_AUDIT_LOG_ENTRIES is set to True (default: False)"
+    )
+
+    def execute(self):
```
Comment (tuomas777) [inline audit_log/management/commands/clear_audit_log_entries.py:16]: Hmm, is this correct, shouldn't it be `handle()`?

Comment (quyenlq) [reply audit_log/management/commands/clear_audit_log_entries.py]: Yes, it should be handle, nice catch

File: audit_log/models.py (hunk 1 above)
Comment (quyenlq) [reply audit_log/models.py]: Good idea, I changed is_sent to sent_at and set the field to DateTimeField

**Review by tuomas777 — APPROVED** [review]
Good stuff! 👍 🚀

**Pull request merged**